content/docs/command-reference/update.md

```diff
@@ -10,7 +10,7 @@ usage: dvc update [-h] [-q | -v] [--rev <commit>] [-R] [--to-remote]
                   [-r <name>] [-j <number>] targets [targets ...]
 
 positional arguments:
-  targets      Import stage .dvc files to update. Using -R, directories
+  targets      Import .dvc files to update. Using -R, directories
```

Comment:
Found this outdated terminology in the `update` ref. only (will check core repo too ⌛). It's "import `.dvc file` everywhere else in docs.